README.md

```diff
@@ -11,7 +11,35 @@ A Flutter package allows you to Showcase/Highlight your widgets step by step.
 
 ![The example app running in Android](https://github.com/simformsolutions/flutter_showcaseview/blob/master/preview/showcaseview.gif)
 
+## Migration guide for release 2.0
+Release 2.0 contains breaking changes in terms of renaming some of the properties from `ShowCaseWidget` and `Showcase` class which is listed below in the table.
 
+<table>
+<tr><th>Before</th><th>After</th></tr>
+<tr><td>autoPlayLockEnable</td><td>enableAutoPlayLock</td></tr>
+<tr><td>shapeBorder</td><td>targetShapeBorder</td></tr>
+<tr><td>showcaseBackgroundColor</td><td>tooltipBackgroundColor</td></tr>
+<tr><td>contentPadding</td><td>tooltipPadding</td></tr>
+<tr><td>overlayPadding</td><td>targetPadding</td></tr>
+<tr><td>radius</td><td>targetBorderRadius</td></tr>
+<tr><td>tipBorderRadius</td><td>tooltipBorderRadius</td></tr>
+</table>
+
+Unused parameters of `Showcase.withWidget()` which are not impacting view of showcase widget that has been removed from parameter list of the named constructor `Showcase.withWidget()`.
```

Comment:
Unused parameters of `Showcase.withWidget()` which do not impact the UI of showcase have been removed from the parameter list of the named constructor `Showcase.withWidget()`.